.bine.json

```diff
@@ -0,0 +1,78 @@
+// See https://github.com/artefactual-labs/bine for config details.
+{
+  "project": "enduro-sdps",
+  "bins": [
+    {
+      "name": "atlas",
+      "version": "0.32.0",
+      "url": "https://release.ariga.io/atlas",
+      "asset_pattern": "atlas-{goos}-{goarch}-v{version}"
+    },
+    {
+      "name": "ent",
+      "go_package": "entgo.io/ent/cmd/ent",
+      "version": "latest" // 0.14.4 still not compatible with go1.24.
```

Comment:
Are you sure that ent 0.14.4 isn't compatible with go 1.24.x? I don't see any errors if I do `make gen-ent` on this branch. :thinking: 

https://github.com/ent/ent/issues/4324 suggests go 1.24.x compatibility was fixed by https://github.com/ent/ent/commit/2b1ad7e4b0128083f50b06f75619c0bea601bdd4 which is included in the ent 0.14.2 ... 1.14.4 tags.

Reply:
You're right! I'm not sure how I came to that conclusion, thanks.